### DISCUSSION TITLE
Can't change both width and spacing

### DISCUSSION THREAD

**Discussion by danbowkley** [supermerill/SuperSlicer#2080] [Q&A]
Under print settings, width&flow, if I change the default flow it automatically changes the width to what it thinks matches the flow value. If I change the width, it changes the flow likewise. My question is, how do I change both of them to what I want? I'm having underextrusion issues and would like to be able to change these settings to match my machine.

I'm using SuperSlicer-ubuntu_18.04-gtk3-2.3.57.7.AppImage on Kubuntu 21.04.

See https://youtu.be/zklIRfxqnmQ for a screen recording.

**Comment by neophyl** [comment]
Simple answer is you can’t. They are linked, you can either change the width and the new spacing is applied to get the correct overlap or vice versa if you need to control the spacing. 

Those settings are not used to adjust for under/over extrusion. Start by confirming your steps per mom is accurate, then measure your filament accurately and set the filament diameter to match reality.  Once those fundamental physical values are accurate only then should you start adjusting other things in the software.
